### ISSUE TITLE
Compare to prev version doesn't work

### ISSUE THREAD

**Issue by codialsoftware** [DonJayamanne/gitHistoryVSCode#320]
... if your repository is checked out to different partition than C. You end up with error message in output: 
```
276221e1e166c71e39ef13a76575eb84677fc680:C:/t:/<pathToFile>  (completed in 0.342s)
fatal: Path 'C:/t:/<pathToFile>' does not exist in '276221e1e166c71e39ef13a76575eb84677fc680'

git log --pretty=oneline 276221e1e166c71e39ef13a76575eb84677fc680^1 -n 1 -- C:\t:\<pathToFile>  (completed in 0.363s)
fatal: Invalid path 'C:/t:': No such file or directory
```

**Comment by stamminator** [comment]
@ole1986 Considering how disruptive this is for the very large userbase that relies on this extension, would you consider doing a hard fork and your own official release of this extension for the VS Code marketplace?

**Comment by ole1986** [comment]
I actually asked Don already to make me a member of the repo to support him...

This hopefully includes releasing new versions

I really hope he is reading my mail soon.

**Comment by Fontenot-Tech** [comment]
Having same issue..just ran into it. Still in the restart/check for updates phase when i happened to come here and see the possible cause. My repo is not on the C: drive either...it's actually on a UNC share (work machine, windows user home share). I'll see if it still doesn't work after checking for updates and if I see the same error in the output.

**Comment by ole1986** [comment]
This issue should be fixed with version 0.4.3
